torchvision/prototype/transforms/functional/_color.py

```diff
@@ -435,7 +435,14 @@ def equalize(inpt: features.InputTypeJIT) -> features.InputTypeJIT:
         return equalize_image_pil(inpt)
 
 
-invert_image_tensor = _FT.invert
+def invert_image_tensor(image: torch.Tensor):
+    num_channels, height, width = get_dimensions_image_tensor(image)
+    if num_channels not in (1, 3):
+        raise TypeError(f"Input image tensor can have 1 or 3 channels, but found {num_channels}")
+
+    return 1.0 - image if image.is_floating_point() else image.bitwise_not()
```

Comment:
The bitwise invert only works for `uint8`, but so does the v1 kernel

https://github.com/pytorch/vision/blob/62da7d48872ef6b9a4a907bbb6b866b39c24120a/torchvision/transforms/functional_tensor.py#L778-L779

Should we maybe add a guard that only accepts floating point and uint8 inputs?

Reply:
Maybe we can keep previous logic and accept all dtypes as previously and make uint8 as a special case ?
Accepting only float and uint8 will be bc breaking

Reply:
If I understand correctly, Victors remarks is in relation to other integer types other than uint8. I'm OK do what he proposes, though I'm not sure if the original method truly handles correctly such `int16` etc, as it assumes their maximum value is 255. No strong opinions on my side.

Reply:
> I'm not sure if the original method truly handles correctly such `int16` etc, as it assumes their maximum value is 255. No strong opinions on my side.

It does not. The behavior is hardcoded for uint8. I'll implement it properly which can be regarded as bug fix.